transforms/angle-brackets/transform.js

```diff
@@ -205,23 +205,32 @@ function transformLinkToAttrs(params) {
     attributes = [firstParamOutput];
   } else if (params.length === 2) {
     // @route and @model param
-
     // eslint-disable-next-line no-unused-vars
     let [_, secondParamInput] = params;
     if (secondParamInput.type === 'SubExpression') {
+      let htmlAttrs;
+      if (secondParamInput.path.original === 'html-attributes') {
```

Comment:
I'm not sure if we should do this only for the second param. The addon works for any param AFAIK. Seems like it also only works for LinkTo here?

Reply:
I agree. I thought it would be a good idea if I included you in the discussion as I worked on it so I could get your feedback. I only had enough time yesterday to tackle a little bit of the problem. 😄